subprojects/jvm-services/src/main/java/org/gradle/internal/jvm/inspection/JvmVendor.java

```diff
@@ -21,16 +21,16 @@
 public interface JvmVendor {
 
     enum KnownJvmVendor {
-        ADOPTIUM("adoptium", "temurin|adoptium|eclipse foundation", "Eclipse Temurin"),
-        ADOPTOPENJDK("adoptopenjdk", "AdoptOpenJDK"),
-        AMAZON("amazon", "Amazon Corretto"),
+        ADOPTIUM("adoptium", "temurin|adoptium|eclipse", "Eclipse Temurin"),
+        ADOPTOPENJDK("adoptopenjdk", "aoj|adoptopenjdk", "AdoptOpenJDK"),
```

Comment:
As discussed, we do not want Foojay Disco APIs mappings making it here.
However, we need to confirm that for the vendors we support, if you have a local JVM, Gradle is able to match it.

There are doubts for some of them.